src/AzureIoTHub.Portal.Shared/Constants/CloudProviders.cs

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) CGI France. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-namespace AzureIoTHub.Portal.Domain.Shared.Constants
+namespace AzureIoTHub.Portal.Shared.Constants
```

Comment:
@delager CloudProviders has been moved from domain layer to dto layer. Is it ok for you ?